### PR TITLE
Card Counting exercise: Replace Utils.feedback with TestedCell

### PR DESCRIPTION
**Pull request by Mraiih**
# Todos
- [x] add `tested_cell` as dependencies
- [x] replace all feedbacks with TestedCell
- [x] add solution for each TestedCell
- [x] remove all assertions in `feedback`

### CODE REVIEW

File: exercises/card_counting.livemd [hunk 1]
```diff
@@ -41,27 +42,63 @@ Card values are as follows:
 When the current card is `4` and the current count is `0` what is the `next_count`?
 Replace `nil` with your answer.
 
+<!-- livebook:{"attrs":{"assertions":"assert next_count === 1","code":"card = 4\r\ncount = 0\r\n\r\nnext_count = nil","solution":"next_count = 1"},"kind":"Elixir.TestedCell","livebook_object":"smart_cell"} -->
```
Comment (BrooklinJazz) [inline exercises/card_counting.livemd:45]: I'd like to encourage students to use arithmetic operators.

Can we change the text to:

When the current card is `4` and the current count is `0` what is the `next_count`?
Use arithmetic operators `+` or `-` with the `count` variable to set the `next_count`.

Then change the solution to:

```elixir
next_count = count + 1
```

File: exercises/card_counting.livemd [hunk 2]
```diff
@@ -41,27 +42,63 @@ Card values are as follows:
...
-
-Utils.feedback(:card_count_four, next_count)
 ```
 
 ### Card King (13)
 
 When the current card is a King `(13)` and the current count is `5` what is the `next_count`?
 Replace `nil` with your answer.
 
+<!-- livebook:{"attrs":{"assertions":"assert next_count === 4","code":"card = 13\r\ncount = 5\r\n\r\nnext_count = nil","solution":"next_count = 4"},"kind":"Elixir.TestedCell","livebook_object":"smart_cell"} -->
```
Comment (BrooklinJazz) [inline exercises/card_counting.livemd:77]: Similar to the above

**Review by BrooklinJazz — APPROVED** [review]
This is awesome! 

This is an improvement, so I'm going to merge the PR. But I would love to make these changes in another PR.

However, after re-reading the exercise, I think we should make some changes. My original explanations could be improved (if you have time to address them in another PR that would be great :) If not I can make them.)

File: exercises/card_counting.livemd (hunk 1 above)
Comment (Mraiih) [reply exercises/card_counting.livemd:45]: Yep I can do that :)

**Pull request merged**